modules/insights-operator-configuring.adoc

```diff
@@ -40,6 +40,7 @@ The *insights-config* `ConfigMap` object follows standard YAML formatting, where
 |`sca: interval`|Specifies the frequency of the simple content access entitlements download.|Time interval|`8h`
 |`sca: disabled`|Disables the simple content access entitlements download.|Boolean|`false`
 |`alerting: disabled`|Disables Insights Operator alerts to the cluster Prometheus instance.|Boolean|`false`
+|`httpProxy`, `httpsProxy`, `noProxy`|Set custom proxy for Insights Operator|URL|`no default`
```

Comment:
Suggestion - if that last cell is meant to say that there is no default value for this field, then maybe you could just state that without using backticks? I'm sure that people would know that the string "no default" is not a valid URL, but technically using backticks does imply that it's a value to use in the file:
```suggestion
|`httpProxy`, `httpsProxy`, `noProxy`|Set custom proxy for Insights Operator|URL|No default
```

Reply:
Hello, @skopacz1 , I see your point. It isn't a value, per se; however, I used the back ticks to make this entry consistent with the others in this column of the table. Because "No default" is not something that a user would enter as a value, as with true or false, or a numeric hour value, I'll go ahead and remove these back ticks. I'll make the change locally and ping you when I've pushed it back here. Thank you for your feedback.